Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: circleci-server-monitoring-stack
```

Comment:
```suggestion
name: cci-server-monitoring-stack
```

Reply:
I feel we should be unambiguous since it can be installed in customer environments.